src/Page/GetHelp.elm

```diff
@@ -15,14 +17,39 @@ view =
             [ h1 [ css [ pageHeadingStyle ] ] [ text (t GetHelpTitle) ]
             ]
         , div [ css [ columnStyle ] ]
-            [ card (t GetHelpSection1Title) (t GetHelpSection1Link) (t SeaSurvivorForumHref)
-            , card (t GetHelpSection2Title) (t GetHelpSection2Link) (t SeaMapFinancialSupportHref)
-            , card (t GetHelpSection3Title) (t GetHelpSection3Link) (t SeaOrganisationsResourceHref)
+            [ card (t GetHelpSection1Title) (t GetHelpSection1Quote1) (t GetHelpSection1Description) JoinForum
+            , card (t GetHelpSection2Title) (t GetHelpSection2Quote1) (t GetHelpSection2Description) CallSupport
+            , card (t GetHelpSection3Title) (t GetHelpSection3Quote1) (t GetHelpSection3Description) SeeOrgs
             ]
         , a [ href (t HelpSelfGridPageSlug) ] [ text (t ToHelpSelfFromGetHelpLink) ]
         ]
 
 
+type CTA
```

Comment:
Nice. `CallToAction` would be more clear But I'll let you get away with it if you want. Chars are free since the build strips all that gubbins out.

Reply:
I'm not a fan of having things that are only differentiated by their casing, I find it harder to read.